### PR TITLE
Project 2: Add and remove items.

### PR DESCRIPTION
**Pull request by Nelitaa**
**Project requirements**
Remove all hardcoded items from the tasks array.
Create a new JavaScript file for the new functionality.
Implement a function for adding a new task (add a new element to the array).
Implement a function for deleting a task (remove an element from the array).
Implement a function for editing task descriptions.
By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..
All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,27 +1,110 @@
+/* eslint-disable class-methods-use-this */
```
Comment (ElsonOtake) [inline src/index.js]: - [ ] Please create a new javascript file for the new functionalities. It's a project requirement and a way of distributing your code for improved readability.

File: src/index.js [hunk 2]
```diff
@@ -1,27 +1,110 @@
...
+    const newTask = new Task(allTasks.tasks.length + 1, inputTask.value);
+    allTasks.addTask(newTask);
+    localStorage.setItem('tasks', JSON.stringify(allTasks.tasks));
+    allTasks.renderTask(newTask);
+    inputTask.value = '';
+  }
+});
+
+tasksList.addEventListener('click', (e) => {
+  if (e.target.classList.contains('edit-task-img')) {
```
Comment (ElsonOtake) [inline src/index.js:23]: - [ ] Kindly fix the functionality of updating a task description. It's not working. When refreshing the screen the old data from local storage is loaded.

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @Nelitaa,

Good job so far! 🎉 🎉  Nice work!! 🏆 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (mgmediaweb) [reply src/index.js]: Good job following the advice of the previous reviewer

File: src/index.js (hunk 2 above)
Comment (mgmediaweb) [reply src/index.js:23]: Good job following the advice of the previous reviewer

**Review by mgmediaweb — APPROVED** [review]
Hi @Nelitaa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved1

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**